build.gradle

```diff
@@ -1944,7 +1944,13 @@ allprojects {
     // By default all of our projects require junit for testing so we can just
     // setup this dependency here.
     dependencies {
-        testImplementation group: "junit", name: "junit", version: "4.8.2"
+        testImplementation group: "junit", name: "junit", version: "4.13.2"
+        testImplementation(group: "org.junit.jupiter", name: "junit-jupiter", version: "5.8.1") {
+            exclude(group: "org.apiguardian", module: "apiguardian-api")
+        }
+        testRuntimeOnly(group: "org.junit.vintage", name: "junit-vintage-engine", version: "5.8.1") {
+            exclude(group: "org.apiguardian", module: "apiguardian-api")
+        }
```

Comment:
As it turns out, `apiguardian` no longer needs to be blocked, so you can restore it if it is easier.

Reply:
Okay, I'll remove the exclusion on apiguardian then as it would just be confusing why it was excluded in the first place.  Hamcrest was already included as JUnit 5 wouldn't work without it.